src/toil/lib/integration.py

```diff
@@ -0,0 +1,322 @@
+import hashlib
```

Comment:
Suggestion: Maybe a docstring for the file explaining what functions should go in "integration".